site/src/components/Footer.tsx

```diff
@@ -81,7 +81,7 @@ const SOCIALS: { name: string; icon: ReactNode; href: string }[] = [
   {
     name: "Discord",
     icon: <Icon className="fab fa-discord" />,
-    href: "https://discord.gg/PefPteFe5j",
+    href: "/discord",
```

Comment:
I expect that this link will not work as the redirects in `next.config.js` are for server-side only. We could either (a) make this absolute or (b) add a `pages/discord.tsx` which handles the redirect client-side.

Reply:
I checked this locally by clicking on the footer link – all worked.
https://blockprotocol-8h51t8v39-hashintel.vercel.app/contact

When we run `next build`, Next creates a catalog of all routes including the redirects, so the client bundle knows how to distinguish a redirect from another page.

If we want Discord link to open in a new tab, I can force it by adding some attributes.

Reply:
82ed24ca8fdeb08eaf94a8396e0113825de2574a

Reply:
Oh, weird - it never used to and [their own docs](https://nextjs.org/docs/api-reference/next.config.js/redirects) still say it doesn't. Maybe a Vercel-specific thing? Apologies!

I think opening in a new tab would be better for anything off-site.

Reply:
> Redirects are only available on the Node.js environment and do not affect client-side routing.

Yeah, that’s a strange statement. Looks like it’s outdated indeed. Just ran `npx create-next-app --example redirects redirects-app` to make sure I’m not crazy – the [example](https://github.com/vercel/next.js/tree/canary/examples/redirects) worked for me (even for internal redirects).

<img width="887" alt="Screenshot 2022-01-12 at 16 29 48" src="https://user-images.githubusercontent.com/608862/149181609-c09ac4ca-a641-49f1-b00c-c72526910577.png">

This does not mean I’m not crazy though 😅